exchange/c2cx/c2cx.go

```diff
@@ -431,3 +431,14 @@ type Orders struct {
 	Orders []Order `json:"orders"`
 	Page   int     `json:"page,omitempty"`
 }
+
+// TickerData includes pricing data for a give currency pair
+type TickerData struct {
+	Timestamp string           `json:"timestamp,omitempty"`
```

Comment:
is Timestamp measured in seconds or milliseconds?  Can you use a custom MarshalJSON and UnmarshalJSON to convert it to a time.Time? (see other examples in this file)